kymatio/scattering1d/frontend/numpy_frontend.py

```diff
@@ -128,7 +128,7 @@ def scattering(self, x):
         signal_shape = x.shape[-1:]
         x = x.reshape((-1, 1) + signal_shape)
 
-        # Precompute output size
+        # Precompute output size  # TODO is this correct? and what's its point?
```

Comment:
this `size_scattering` thing comes from `Scattering1D`. I guess that the rationale is that it's easier on the garbage collector to pre-allocate the output once rather than concatenate it all at the end. But i'm not sure how useful it is (nor how well we abide to this idea in `Scattering1D`)
it's essentially an optimization aspect and shouldn't affect the result